deprecation.py

```diff
@@ -121,10 +124,10 @@ def deprecated(deprecated_in=None, removed_in=None, current_version=None,
                           means immediate deprecation. If this is not
                           specified, then the `removed_in` and
                           `current_version` arguments are ignored.
-    :param removed_in: The version when the decorated method will be removed.
-                       The default is **None**, specifying that the function
-                       is not currently planned to be removed.
-                       Note: This cannot be set to a value if
+    :param removed_in: The version or datetime.date when the decorated method
```

Comment:
Similar to above, `` :class:`datetime.date` `` gets us nice formatting/linking in the docs.

Reply:
I've made the change